charts/fluid-datamigrate/juicefs/templates/configmap.yaml

```diff
@@ -13,16 +13,26 @@ data:
     function main() {
       echo "juicefs datamigrate job start..."
 
-      if [ $EDITION == 'community' ]
+      # handle parallel migrations
+      if [ $PARALLELISM -gt 1 ]
       then
-        timeout $TIMEOUT /usr/local/bin/juicefs sync {{ .Values.datamigrate.migrateFrom }} {{ .Values.datamigrate.migrateTo }} $OPTION
+        workers=$(grep "Host " /root/.ssh/config  | cut -d " " -f 2  | tr '\n' ',')
```

Comment:
I think only the enterprise version support multiple workers sync. please help confirm @zwwhdls

Reply:
The community release also supports it.